pages.it/windows/ftp.md

```diff
@@ -0,0 +1,36 @@
+# ftp
+
+> Interactively transfer files between a local and remote FTP server.
+> More information: <https://learn.microsoft.com/windows-server/administration/windows-commands/ftp>.
+
+- Connettiti a un server FTP remoto:
+
+`ftp {{host}}`
+
+- Accedi come utente anonimo:
+
+`ftp -A {{host}}`
+
+- Disabilita l'accesso automatico alla connessione iniziale:
+
+`ftp -n {{host}}`
+
+- Esegui un file contenente una lista di comandi FTP:
+
+`ftp -s:{{percorso\to\file}} {{host}}`
+
+- Scarica file:
+
+`mget {{*.png}}`
+
+- Carica file:
```

Comment:
I would suggest mentioning the glob expression's equivalent term here.

Ref.
https://github.com/tldr-pages/tldr/blob/bead5b84583096d38b5da425fbe3ff0ba744da5a/pages/windows/ftp.md?plain=1#L26